test-in-k8s.yaml

```diff
@@ -76,6 +76,9 @@ rules:
 - apiGroups: [""]
   resources: ["pods/log"]
   verbs: ["get","list","watch"]
+- apiGroups: [""]
+  resources: ["events"]
+  verbs: ["watch"]
```

Comment:
What happens if this permission is not granted? Does the plugin gracefully ignore the failure? Otherwise we are creating a minor backward compatibility issue. For example, CloudBees Core does not currently seem to grant this.

Reply:
I noted it. Doable in theory through `SelfSubjectAccessReview` unfortunately support for this object kind seems to be broken in kubernetes-client at the moment.

Reply:
Fixed.